src/util/address.rs

```diff
@@ -907,8 +954,8 @@ mod tests {
             ("bc1p0xlxvlhemja6c4dqv22uapctqupfhlxm9h8z3k2e72q4k9hcz7vqzk5jj0", "512079be667ef9dcbbac55a06295ce870b07029bfcdb2dce28d959f2815b16f81798")
         ];
         for vector in &valid_vectors {
-            let addr: Address = vector.0.parse().unwrap();
-            assert_eq!(&addr.script_pubkey().as_bytes().to_hex(), vector.1);
+            let addr: Address = vector.0.parse().expect(&format!("error in test vector {}", vector.0));
```

Comment:
TIL: non-static strings are allowed in `expect`. Although I'd avoid this in production code as it always has to do the string interpolation, not only in the error case. Ok for tests though imo.